commands/apps_dev.go

```diff
@@ -269,8 +293,11 @@ func RunAppsDevBuild(c *CmdConfig) error {
 	// }
 
 	buildingComponentLine := template.String(
-		`building {{lower (snakeToTitle .GetType)}} {{highlight .GetName}}`,
-		componentSpec,
+		`building {{lower (snakeToTitle .componentSpec.GetType)}} {{highlight .componentSpec.GetName}} {{muted print "(" .appName ")"}}`,
```

Comment:
muted takes only 1 arg and this will cause a failure

Reply:
fixed